docs/pages/server-access/architecture.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Teleport Server Access Architecture Overview
+description: Surveying key concepts and architectural concerns when using Teleport Server Access.
+h1: Teleport Server Access
+---
+
+Many administrators will consider several factors when configuring Teleport for their specific stack, architecture, and resources.
+
+This document surveys key concepts and architectural concerns when using Teleport Server Access.
+
+## Choose your SSH solution
```

Comment:
This section will likely be cut and moved into https://github.com/gravitational/teleport/pull/7043

Some of the security topics can be moved into the planned Security Architecture Guide.